deploy/action.yaml

```diff
@@ -141,3 +141,26 @@ runs:
         --set ${{ inputs.tag_value_path }}=${{ inputs.image_tag }}  \
         --timeout ${{ inputs.helm_timeout }}                        \
         ${{ inputs.helm_values }}
+
+    # List installed charts after upgrade
+    - name: helm list
+      shell: bash
+      run: helm list -n ${{ inputs.app_name }}-${{ inputs.env_name }} -a
```

Comment:
Do you want to filter here or list all the charts?

Reply:
(might need an `if: always()`, to also print versions after a failed deployment)